### DISCUSSION TITLE
Write out banned.cfg to config folder

### DISCUSSION THREAD

**Discussion by cs-deathmatch** [JamesIves/hlds-docker#39] [Ideas and Feature Requests] [answered]
When using AMX to ban a player, it stores their information in banned.cfg. but when the server restarts, this information is lost. Whenever the server modifies banned.cfg, it should update the file stored in the config folder

**Comment by JamesIves** [comment]
This doesn't seem like intended behavior. Couple of questions to help debug:

1. Where does banned.cfg get written to?
2. Is it just the contents of the file that are stripped out, or the entire file?
3. What command are you using to start the container?

**Comment by cs-deathmatch** [comment, marked as answer]
whoops, forgot to put exec banned.cfg in my server config 🤦‍♀️
